packages/notification-store/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## [1.0.1] - 2019-03-18
+- First release
```

Comment:
Isn't it weird that 1.0.1 is the 'first release'? Maybe you should also list 1.0.0 as a broken release?

Reply:
Since the previous versions are all alpha versions, this would be the first production release.
I'm guessing the original reason behind this version number is because there's a previous version on Bitbucket.
Perhaps I should link to the Bitbucket version as first release?

Reply:
No, I would only list `1.0.0` as not working. No references to Bitbucket plz.